scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -1,12 +1,23 @@
 ---
-title: "scRNA-seq Dimension Reduction"
-output:   
+title: "Dimension Reduction with scRNA-seq data"
+author: CCDL for ALSF
+date: 2021
+output:
   html_notebook: 
     toc: true
     toc_float: true
 ---
 
-**CCDL 2020**
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Read Cell Ranger data into R 
+- Filter post-quantification cells using `emptyDrops()`
+- Apply dimensionality reduction methods to single cell data
+- Visualize reduced dimensionality data
```

Comment:
I haven't read further, but I assume that this is what we're doing and I think being a bit more specific could be helpful:

```suggestion
- Visualize samples in reduced dimensional space
```